### PR TITLE
Persist data into a JSON file PR

### PR DESCRIPTION
**Pull request by Aseadd**
#### Project requirements
For this PR the following requirements were adhered to:

- [x] Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).

- [x]  On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly

- [x] On startup of your app load all data from files

### CODE REVIEW

File: src/persist_book.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+require 'json'
+
+class PersistBook
+  def initialize(book)
+    @books = book
+  end
+  include List
+
+  def read_book_data
+    book_data = File.read('./src/books.json')
```
Comment (omarsalem7) [inline src/persist_book.rb:15]: - [ ] Kindly handle the case when one or more of the files are missing (the app should work and show no errors). By adding this line: ` return [] unless File.exists?("./src/books.json")` for all files to check file exists or not.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Aseadd 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/persist_book.rb (hunk 1 above)
Comment (Aseadd) [reply src/persist_book.rb:15]: make the changes accordingly, Thank you for the valuable comment

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Aseadd, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**